op-e2e/actions/helpers/user.go

```diff
@@ -204,6 +204,12 @@ func (s *BasicUser[B]) ActSetTxValue(value *big.Int) Action {
 	}
 }
 
+func (s *BasicUser[B]) ActSetGasLimit(limit uint64) Action {
```

Comment:
```suggestion
func (s *BasicUser[B]) ActSetTxGasLimit(limit uint64) Action {
```

slightly more consistent with other setters